src/shared/constants.h

```diff
@@ -82,3 +82,24 @@ const double POSSESSION_TIMESTAMP_TOLERANCE_IN_MILLISECONDS = 10;
  */
 const double ANGLE_TO_ROBOT_FRONT_WHEELS_DEG = 57.945;
 const double ANGLE_TO_ROBOT_BACK_WHEELS_DEG  = 136.04;
+
+
+// Networking
+// the IPv6 multicast address, only ff02 is important, the rest is random
+// see https://en.wikipedia.org/wiki/Solicited-node_multicast_address for why ff02 matters
+// TODO is there a better way to do this? w/out doing string manip
+const char MULTICAST_CHANNELS[16][21] = {
```

Comment:
Is there a reason why it's defined as 16x21?

Reply:
its the size of the channels, we have 16 groups and each group is 21 characters long

we can't use constants here instead of 16/21 because its a const char array

Reply:
could we at least create these as `#define`'s?

Reply:
fixed